src/BuildIt.Forms/BuildIt.Forms.Controls/Platforms/Ios/CameraPreviewControlRenderer.cs

```diff
@@ -128,6 +128,11 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
             {
                 EnableContinuousAutofocus(cameraPreviewControl.EnableContinuousAutoFocus);
             }
+
+            if (e.PropertyName == CameraPreviewControl.AspectProperty.PropertyName)
```

Comment:
Would it better to use nameof(Aspect) here?

Reply:
No IMO as this creates a better coupling. Using nameof(Aspect) assumes that the bindable property was created with nameof(Aspect) as the name. Generally we do that but it in theory could be something else and If the value got changed then a change would also need be changed here

Reply:
but that logic doesn't hold since you are using nameof(Aspect) when defining the property. nameof gets compiled out so would be more efficient than looking at PropertyName at runtime.

Reply:
I know that's why I mentioned in theory to point out using PropertyName more accurately reflects the relationship, bearing in mind that is not just about efficiency